api/src/pcapi/core/search/backends/algolia.py

```diff
@@ -281,6 +281,7 @@ def serialize_offer(cls, offer: offers_models.Offer) -> dict:
                 "thumbUrl": url_path(offer.thumbUrl),
                 "tags": tags,
                 "times": list(set(times)),
+                "students": offer.extraData["students"] if offer.extraData and offer.extraData["students"] else [],
```

Comment:
```suggestion
                "students": extra_data.get("students") or [],  # ou `.get("students", [])` selon l'effet recherché
```

On pourrait aussi remonter l'attribut quelques lignes au-dessus pour conserver l'ordre alphabétique. ;)